Python/Product/Analysis/Interpreter/Ast/AstPythonInterpreterFactory.cs

```diff
@@ -125,15 +125,7 @@ InterpreterFactoryCreationOptions options
             }
 
             if (File.Exists(_searchPathCachePath)) {
-                for (int retries = 5; retries > 0; --retries) {
-                    try {
-                        File.Delete(_searchPathCachePath);
-                        break;
-                    } catch (IOException) {
-                    } catch (UnauthorizedAccessException) {
-                    }
-                    Thread.Sleep(10);
-                }
+                PathUtils.DeleteFile(_searchPathCachePath);
```

Comment:
Don't you call `File.Exists` in `PathUtils.DeleteFile`?

Reply:
No, it will retry and give up without an error, but it doesn't check.